scaleway/resource_lb_frontend.go

```diff
@@ -211,9 +211,9 @@ func resourceScalewayLbFrontendRead(ctx context.Context, d *schema.ResourceData,
 		return diag.FromErr(err)
 	}
 
-	_, lbID, err := parseZonedID(d.Get("lb_id").(string))
-	if err != nil {
-		return diag.FromErr(err)
+	lbID := expandID(d.Get("lb_id"))
+	if len(lbID) == 0 {
```

Comment:
Why should it be a length list? why not compare it to an empty string?

Reply:
Just changed